src/models/db.js

```diff
@@ -84,6 +95,13 @@ export default config => {
     }
   });
 
+  hooks.forEach(hook => {
+    const clientHook = config.hooks[hook];
+    if (clientHook) {
+      sequelize.addHook(hook, clientHook);
+    }
+  });
+
```

Comment:
Aha, somehow I didn't pick up on that approach from the documentation. I agree it is the best approach (most concise).